server/src/HttpServer.hs

```diff
@@ -21,17 +23,20 @@ new core =
       middleware $ jwtMiddleware $ configJwtSecret $ coreConfig core
 
     get (regex "^") $ do
+      liftIO $ forM_ (coreMetrics core) Metrics.notifyGetRequest
```

Comment:
Can you abstract this to something like:

``` haskell
updateMetrics :: Metrics.Notification -> ?
updateMetrics = liftIO . forM_ (coreMetrics core) 

-- and then:
put (regex "^") $ do
       updateMetrics Metrics.notifyPutRequest
--etc

The main point is that we want it to read high-level, like prose. 
```

Reply:
Done, does indeed look nicer now.

Reply:
Nice :+1:. I find this to be a cornerstone of my programming culture: to make code read as prose.